doc/dev/decisionrecords/DevelopmentProcessConventions.md

```diff
@@ -31,29 +31,39 @@ Time boxes:
 
-#### 3. Only discuss draft PRs and issues in dev-meeting if they are on the agenda
+### 3. Only discuss draft PRs and issues in dev-meeting if they are on the agenda
 To save time, we do not want to go through every issue and draft PR every dev-meeting. If a discussion on a specific draft PR or issue is needed, then it should be put onto the agenda.
 
-#### 4. Use threads in gitter
+### 4. Use threads in gitter
 To avoid clutter in our Gitter channel, always use threads. Messages not abiding by this rule will be deleted.
 
-#### 5. Decide *early* if we want to invest resources into a PR made by non-core developers
+### 5. Decide *early* if we want to invest resources into a PR made by non-core developers
 All PRs made by non-core developers lead to the core developers having to spend resources on reviewing the code and potentially having to provide guidance to the author. We need to decide early, if the PR will be beneficial to enough contributors that investing these resources is worth it. This is especially important for PRs that will touch the core.
 
 Product Owners should have the ultimate decision in this. That way the core developers will have the official backing of their employers and have fewer conflicts investing their time.
 
-#### 6. Issues not made by regular contributors will have to be categorized by adding a label
+### 6. Issues will have to be categorized by setting a type
+Every PR that isn’t a minor change should have a connected issue. 
 
-To be able to deal with issues efficiently, issues will have to be categorized. Markus will go through new issues:
-1. Close it if it is not a requested change to OTP
-2. Assign one of the following labels:
- - `Bug`
- - `Documentation`
- - `Improvement`
- - `New Feature`
- - `Optimization`
- - `Technical Debt`
+All issues describing modifications to OTP need to be one of these types:
+ - `Bug` Un unexpected problem or behaviour.
+ - `Feature` A request, idea, or new functionality.
+ - `Technical Improvement` None functional improvements like refactoring, optimizations, cleanup and CI/CD.
 
+In case it's a minor change we set this label on the PR: `Minor Change`. The definition of something being a minor change is that there’s no need for preparing the change by reviewing the desing or the modification of OTP functionality. Documentation is often a minor change.
+
+
+### 7. Approval of the different types of OTP modifications
+For changes classified as `Bug` or `Technical Improvement`, approval of the architectural design is considered sufficient.
+
+For changes classified as `Feature`, both the approval of that it is a desired change in OTP, and approval of the architectural design are required.
+
+Tags we use for approval:
+- `Design approved` Arcitectual design approved.
+- `Feature approved` New functionality approved.
+- `Feature rejected` New functionality rejected.
```

Comment:
In case an issue or PR is rejected (this is the same process for both):
 - Close issue with a reason why it is closed and set milestone to `rejected`.  
 - No label - it is redundant